namespace java dvr.netsdk
namespace cpp  dvr.netsdk

service NetSdk{
	
}